src/build.py

```diff
@@ -1233,15 +1241,15 @@ def Summary(repos):
       buildbot.Link('lkgr.json', cloud.Upload(info_file, lkgr_file))
 
 
-def AllBuilds(use_asm=False):
+def AllBuilds(use_asm=False, run_tests=False):
```

Comment:
Why not just pass in options here, and have sub-tests look at run_tool_tests if they want to?